config/application.yml.default

```diff
@@ -98,6 +98,7 @@ otp_valid_for: '10'
 otps_per_ip_limit: '25'
 otps_per_ip_period: '300'
 otps_per_ip_track_only_mode: 'true'
+outbound_connection_check_url: 'https://checkip.amazonaws.com'
```

Comment:
this URL was a suggestion from devops team

Reply:
Is there any interest in making this configurable via URL param? Similar to how our PKI certs health controller lets us specify different lengths of time? That way we could set up checks against multiple destinations if we wanted to?

Can also add that functionality later if we need to

Reply:
I think hard-coded is fine for now. We'd have to make it an allow list or something.

Reply:
I also think hard-coded is fine for now. If we do end up going with a url param we should definitely make sure we have an allowlist.